internal/rbd/nodeserver.go

```diff
@@ -460,6 +472,11 @@ func (ns *NodeServer) mountVolumeToStagePath(ctx context.Context, req *csi.NodeS
 			args = []string{"-m0", "-Enodiscard,lazy_itable_init=1,lazy_journal_init=1", devicePath}
 		} else if fsType == "xfs" {
 			args = []string{"-K", devicePath}
+			// always disable reflink
+			// TODO: make enabling an option, see ceph/ceph-csi#1256
+			if ns.xfsSupportsReflink() {
+				args = append(args, "-m", "reflink=0")
```

Comment:
Define a new variable to store 0 or 1 , it will help when we want to re-enable it

Reply:
There is no logic to enable it yes, so I do not see why this PR should make it configurable.